crates/api_models/src/payments.rs

```diff
@@ -1516,6 +1517,34 @@ pub struct PaymentListResponse {
     pub data: Vec<PaymentsResponse>,
 }
 
+#[derive(Clone, Debug, serde::Serialize, ToSchema)]
+pub struct PaymentListFilters {
```

Comment:
The name seems to indicate that this is a filter, but it seems to be a response based on serde::Serialize derive. Can you change the name accordingly to indicate that it is a response?

Reply:
Ok what name should it be